addons/vagrant/inventory/group_vars/pfservers/venom.yml

```diff
@@ -1,4 +1,4 @@
 ---
-venom__version: 'v1.0.0-rc.7'
+venom__version: 'v1.0.0'
```

Comment:
We will have to use latest stable version from v1.0.X

Reply:
Updated